src/gpb.erl

```diff
@@ -34,8 +34,8 @@
 -export([defs_records_to_proplists/1,  proplists_to_defs_records/1]).
 -export([rpc_records_to_proplists/1, rpc_record_to_proplist/1, proplists_to_rpc_records/1]).
 -include_lib("eunit/include/eunit.hrl").
--include("../include/gpb.hrl").
--include("../include/gpb_version.hrl").
+-include("gpb.hrl").
```

Comment:
What is the purpose of the change to remove `"../include"` from `-include` lines?
This change breaks when compiling from within Emacs, which I do all the time. Does this change make things better in some other end?

 I suppose rebar automatically adds the `"include"` path to the compilation options, but with an unmodified Emacs setup, I this is not the case. I don't know about other editors, such as vi, but for emacs, it is possible to set up a [`.dir-locals.el`](https://www.gnu.org/software/emacs/manual/html_node/elisp/Directory-Local-Variables.html) file at the top, but if it does not hurt having those `"../include"` paths, I would prefer this change to be a separate PR.

(this comment applies not only to this line, but to several others)